src/OSPSuite.SimModel/src/SimulationOptions.cpp

```diff
@@ -15,13 +15,14 @@ using namespace std;
 
 SimulationOptions::SimulationOptions(void)
 {
-	_validateXMLWithSchema = true;
+	_validateXMLWithSchema = false; //perform schema validation only if explicitly set by user
 	_showProgress = false;
 	_executionTimeLimit = 0.0;    //no limit of simulation execution time
 	_stopOnWarnings = true;       //treat warnings of ODE solver as errors
 	_autoReduceTolerances = true; //automatically reduce solver tolerances
 	_writeLogFile = true;
 	_checkForNegativeValues = true;
+	_identifyUsedParameters = false;
```

Comment:
Not sure if we need the flag or if used parameters should be ALWAYS estimated.
Currently there is a posiibility to skip this step (in this case, ALL parameters will be set as used)